app/views/users/edit.html.haml

```diff
@@ -0,0 +1,29 @@
+= content_for :head_content do
+  :javascript
+    $(document).ready(function() {
+      $("input#user_email").keyup(function(e) {
+        $("input#user_username").val(this.value);
+      });
+
+      $("input#user_email").change(function(e) {
+        $("input#user_username").val(this.value);
+      });
+    });
+
+= content_for :h1 do
+  =h current_facility
```

Comment:
We shouldn't be using the `h` anymore. That's a pre-rails 3 vestige that should go away wherever we find it.